src/interfaces/assistants_web/src/components/Agents/AgentSettings/AgentSettingsForm.tsx

```diff
@@ -142,7 +145,11 @@ export const AgentSettingsForm: React.FC<Props> = ({
         isExpanded={currentStep === 'define'}
         setIsExpanded={(expanded) => setCurrentStep(expanded ? 'define' : undefined)}
       >
-        <DefineAssistantStep fields={fields} setFields={setFields} nameError={nameError} />
+        <DefineAssistantStep
+          fields={fields}
+          setFields={setFields}
+          nameError={!!fields.name ? nameError : undefined}
```

Comment:
`fields.name` is required - if we don't want to show the 'Assistant name is required' error message, we can just remove from `nameError`
```suggestion
          nameError={nameError}
```

Reply:
i think showing it at the beginning it is too annoying, might remove it altogether!